docs/Deep-Dives/metaapi.md

```diff
@@ -0,0 +1,200 @@
+title: Flotiq and Meta-API integration | Flotiq docs
+description: This tutorial shows how to use Flotiq and Meta-API to integrate with another API
+
+
+# Flotiq + Meta-API + Google Spreadsheets use case
+
+This guide shows how to add your private Flotiq API to Meta-API and prepare an example Spell. 
+In our case we will create a report in Google Spreadsheet based on products stock quantity.
+
+![Integrate Flotiq with Google Spreadsheets using Meta-API Spell](images/metaapi/img-0.png){: .border}
+
+
+## 1. Create an empty sheet in Google Spreadsheets
+
+Let's prepare our spreadsheet to use in further steps. 
+We add two columns (Date, Stock Quantity) and a chart with the default config for these columns.
+
+![Create an empty sheet to store Flotiq aggregated data](images/metaapi/meta-spr-1.png){: .border}
+
+Notice the `spreadsheet id`. 
```

Comment:
We need to find the spreadsheet id
It can be found inside the spreadsheet link...
<spreadsheet_id>
Save the id, we will need it in future steps.

Reply:
I really suggest you to change [__here__] to e.g <spreadsheet_id> its more readable in my opinion